crates/compiler/parse/src/blankspace.rs

```diff
@@ -29,6 +29,26 @@ where
     )
 }
 
+pub fn space0_around_e_no_after_indent_check<'a, P, S, E>(
+    parser: P,
+    indent_before_problem: fn(Position) -> E,
+) -> impl Parser<'a, Loc<S>, E>
+where
+    S: Spaceable<'a>,
+    S: 'a,
```

Comment:
Any reason not to combine the bounds?
```suggestion
    S: Spaceable<'a> + 'a,
```